src/app/operators/dialogs/numeric-attribute-filter/numeric-attribute-filter.component.html

```diff
@@ -36,14 +36,17 @@
             <mat-form-field class="flex-item" fxFlex>
                 <input matInput type="number" step="any" placeholder="Min" formControlName="min">
                 <span matSuffix>{{attributeUnit$ | async}}</span>
-                <mat-hint class="error" *ngIf="form.controls.bounds.errors?.noFilter">
-                    Specify at least minimum or maximum.
+                <mat-hint [ngClass]="{'mat-error': true, 'mat-hint': false}" *ngIf="form.controls.bounds.errors?.noFilter">Specify at least
+                    minimum or maximum.
+                </mat-hint>
+                <mat-hint [ngClass]="{'mat-error': true, 'mat-hint': false}" *ngIf="form.controls.bounds.errors?.noFiniteNumber">The numbers
+                    must be valid and finite.
```

Comment:
Kann es passieren, dass beide Fälle eintreten? Das führt soweit ich das kenne zu einer Exception.

Reply:
In 05c2030 gibt es nur noch eins, das andere ist es nie aufgetreten, da das Zahlenfeld die Nicht-Zahlen gar nicht durchgibt.